crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1192,10 +1192,9 @@ fn visit_node(
             let mut font_features = Vec::new();
             for (flag, value) in flags {
                 let flag_ascii = flag.to_ascii_lowercase();
-                let flag_bytes = flag_ascii.as_bytes();
-                if flag_bytes.len() == 4 {
-                    let tag = [flag_bytes[0], flag_bytes[1], flag_bytes[2], flag_bytes[3]];
-                    font_features.push(FontFeature { tag, enabled: *value == 1 });
+                if flag_ascii.len() == 4 {
+                    // Smoke check to see if the flag is valid
```

Comment:
What is this "Smoke check" referring to?

Reply:
A bit of crude validation. In this case validating that the flag is exactly 4 characters.